index.html

```diff
@@ -185,15 +303,16 @@ <h1 class="greet">
           <li class="angelist"><a href="#"></a><img src="assets/Vector.png"></li>    
         </ul>
         <div class="resume">
-          <button class="button" id="resume">Get my resume</button>
+          <button type="button" class="button" id="resume">Get my resume</button>
         </div>
-
-        
+      </div>
+        <div class="grid_item2">
           <ul class="skills">
-              <ul class="languages">  
+              <ul class="languages"> 
                 <li><p>Languages</p></li>
-                <li><img src="assets/Unionlangs.png"></li>
+                <li class="arrow"><img src="assets/Unionlangs.png"></li>
```

Comment:
- [ ] You should stick to the design as much as possible. Please increase the font weight of the heading, and adjust the text to match the Figma design. On the right side the drop-down menu is missing please implement it and also line after the framework and skill are missing please do so. See the image below for reference.
- [ ] Also fix the padding from the left and right of buttons and also the text size of the buttons.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213866808-31d4870d-ddef-4de5-b77d-1bbd017f48ca.jpg)

### Figma design

![fimga](https://user-images.githubusercontent.com/81727988/213866814-6a10de85-4471-4f6c-a73a-21c077e53dca.jpg)